docs/support/prometheus.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: default
+title: Prometheus
+parent: Supporting Applications
+nav_order: 750
+---
+## Prometheus: System Monitoring
+
+There's a lot going on within the BitBox Base and it's good to have reliable metrics over time to aid in development, fine-tuning, during operations and when looking for the cause of a specific error.
+[Prometheus](https://prometheus.io/) takes care of that.
+It is a open-source monitoring solution that is easily extendable to monitor any application.
+Measurements are stored in a time-series database that can easily be queried and visualized, e.g. using [Grafana](grafana.md).
+This database is also the main data pool that is queried by the Base Middleware.
+
+### Installation
+
+The Prometheus archive file is downloaded directly from the GitHub releases page and checked against a hardcoded checksum.
```

Comment:
Maybe it's just me, but "archive file" sounds more like it's about the data used to configure Prometheus, whereas we are describing the application itself here. Perhaps just "Prometheus is downloaded directly from the GitHub releases page [...]"?

Reply:
ACK